Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
Is this added by a new version of Cargo?

Reply:
> Is this added by a new version of Cargo?

Yes: https://github.com/rust-lang/cargo/pull/6548